src/install/config/dotnet.yaml

```diff
@@ -217,14 +217,28 @@ steps:
           - optionType: deployment
             options:
               - value: 'awsLambda'
-  - filePath: 'src/install/dotnet/installation/azure-web-apps.mdx'
+  - optionType: deployment 
```

Comment:
So this is actually in the wrong place. The branching doc is divided into two main sections:
- `Appinfo`
-  `Steps`

You need to move this entire section up until line 235 to the `Appinfo` section at the top of this doc.